manifests/client.pp

```diff
@@ -142,21 +142,22 @@
 #        }
 class xymon::client (
   String $xymon_server,
-  Boolean $manage_repository            = true,
-  Boolean $include_clientlaunch_d       = false,
-  String $config_file                   = '/etc/default/xymon-client',
-  String $package                       = 'xymon-client',
-  String $service_name                  = 'xymon-client',
-  String $xymon_config_dir              = '/etc/xymon',
-  String $xymon_user                    = 'xymon',
-  String $xymon_group                   = 'xymon',
-  Optional[String] $repository_url      = undef,
-  Optional[String] $gpg_url             = undef,
-  Optional[String] $gpg_id              = undef,
-  Optional[Hash] $monitors              = undef,
-  Optional[String] $client_name         = undef,
-  Optional[String] $clientlaunch_config = undef,
-  Optional[String] $files_path          = undef,
+  Enum['present', 'latest', 'absent'] $package_ensure = 'latest',
```

Comment:
Will approve, but shouldn't we allow a version number here, if someone (even we?) want to stick the package to a specific version?

Reply:
Hm. Right. Then I just set it to string and let the package type do the checking.

Reply:
Fixed